operator/v1/types.go

```diff
@@ -70,11 +70,12 @@ type OperatorSpec struct {
 	// +kubebuilder:default=Normal
 	OperatorLogLevel LogLevel `json:"operatorLogLevel,omitempty"`
 
-	// unsupportedConfigOverrides holds a sparse config that will override any previously set options.  It only needs to be the fields to override
-	// it will end up overlaying in the following order:
-	// 1. hardcoded defaults
-	// 2. observedConfig
-	// 3. unsupportedConfigOverrides
+	// unsupportedConfigOverrides overrides the final configuration that was computed by the operator.
+	// By using this property, you create an environment where configuration properties can be overwritten or added in a way that is not supported by Red Hat.  
```

Comment:
This sentence is a bit long, do we need it to be so verbose? I think all we need to say here is that using this is unsupported right?
```suggestion
	// Use of this field is not supported by Red Hat.  
```

Reply:
Agreed, @JoelSpeed . I updated the content to make it shorter.